Zend/zend_API.h

```diff
@@ -1794,9 +1794,9 @@ ZEND_API ZEND_COLD void zend_argument_value_error(uint32_t arg_num, const char *
 	Z_PARAM_DOUBLE_EX(dest, is_null, 1, 0)
 
 /* old "f" */
-#define Z_PARAM_FUNC_EX(dest_fci, dest_fcc, check_null, deref) \
+#define Z_PARAM_FUNC_EX(dest_fci, dest_fcc, check_null, deref, free_trampoline) \
```

Comment:
This may break third-party extensions.

e.g. pecl/memcached https://github.com/dstogov/php-src/actions/runs/6483064666/job/17603831990

I think it's better to keep ``Z_PARAM_FUNC_EX`` unchanged and add ``Z_PARAM_FUNC_EX2``.

cc: @iluuu1994 @nielsdos

Reply:
I'm on it: #12419